### PR TITLE
minimize step and server image

### PR DESCRIPTION
**Pull request by lhw362950217**
Fix #2526 this PR cut server image from 1.1G to 550M, step image from 2.1G to 1.4G

### CODE REVIEW

File: docker/step/Dockerfile [hunk 1]
```diff
@@ -11,35 +11,56 @@ RUN /bin/bash -c 'source find_fastest_resources.sh \
...
 ADD build/*.jar /opt/sqlflow/parser/
 ADD python/sqlflow_submitter /opt/sqlflow/python/sqlflow_submitter/
 ADD python/symbol_extractor.py /opt/sqlflow/python/
 ADD python/plotille_text_backend.py /opt/sqlflow/python/
-ADD build/models/sqlflow_models /opt/sqlflow/python/sqlflow_models/
 
-ENV PYTHONPATH /opt/sqlflow/python
+
+ENV PATH "${PATH}:/usr/local/odpscmd/bin"
+ENV PYTHONPATH "${PYTHONPATH}:/usr/lib/python3.6/site-packages:/opt/sqlflow/python"
```
Comment (Yancey0623) [inline docker/step/Dockerfile:65]: `/usr/lib/python3.6/site-packages` is the global site-packages directory, maybe we don't need to add this in `$PYTHONPATH` ?

File: docker/server/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
         && echo "Choose the fastest PIP source ..." \
         && choose_fastest_pip_source'
 
 # Install Python and JRE.  SQLFlow server requires Couler/Fluid to generate YAML
 # files, and Couler/Fluid depends on Python.  The external parsers are in Java.
-RUN apt-get -qq install -y \
-           python3-dev python3-pip openjdk-8-jre-headless > /dev/null \
-        && rm -rf /var/lib/apt/lists/* \
-        && apt-get -qq clean -y
+RUN apk update \
```
Comment (typhoonzero) [inline docker/server/Dockerfile:15]: Can put all `RUN` in one line to reduce the Docker image layers.

File: docker/step/Dockerfile (hunk 1 above)
Comment (lhw362950217) [reply docker/step/Dockerfile:65]: It seems that this does not work automatically, I haven't figured out the reason.

File: docker/server/Dockerfile (hunk 2 above)
Comment (lhw362950217) [reply docker/server/Dockerfile:15]: Done, I put some relative RUN together

**Review by typhoonzero — APPROVED** [review]

**Review by wangkuiyi — APPROVED** [review]
LGTM++

**Pull request merged**